libraries/Microsoft.Bot.Builder.Dialogs/SkillDialog.cs

```diff
@@ -47,10 +47,10 @@ public override async Task<DialogTurnResult> BeginDialogAsync(DialogContext dc,
 
             // Store delivery mode and connection name in dialog state for later use.
             dc.ActiveDialog.State[DeliverModeStateKey] = dialogArgs.Activity.DeliveryMode;
-            dc.ActiveDialog.State[SsoConnectionNameKey] = dialogArgs.ConnectionName;
+            dc.ActiveDialog.State[SsoConnectionNameKey] = DialogOptions.ConnectionName;
```

Comment:
We can remove the storing of the connectionName in the DialogState because the connectionName is passed into the SkillDialog during construction.

Reply:
Removed.